test/test_shallow_water_models.jl

```diff
@@ -3,6 +3,8 @@ include("dependencies_for_runtests.jl")
 using Oceananigans.Models.ShallowWaterModels
 using Oceananigans.ImmersedBoundaries: ImmersedBoundaryGrid, GridFittedBoundary
 
+CUDA.allowscalar(true)
+
```

Comment:
why this?

Reply:
To run it on a GPU I needed to add this to work.  But I haven't tested it and this runs on a CPU so I"m happy to delete it if you prefer.  Probably better to remove it.

Reply:
I removed this line.